internal/pkg/db/mongo/models/value_descriptor.go

```diff
@@ -20,20 +20,22 @@ import (
 )
 
 type ValueDescriptor struct {
-	Id           bson.ObjectId `bson:"_id,omitempty"`
-	Uuid         string        `bson:"uuid"`
-	Created      int64         `bson:"created"`
-	Description  string        `bson:"description,omitempty"`
-	Modified     int64         `bson:"modified"`
-	Origin       int64         `bson:"origin"`
-	Name         string        `bson:"name"`
-	Min          interface{}   `bson:"min,omitempty"`
-	Max          interface{}   `bson:"max,omitempty"`
-	DefaultValue interface{}   `bson:"defaultValue,omitempty"`
-	Type         string        `bson:"type,omitempty"`
-	UomLabel     string        `bson:"uomLabel,omitempty"`
-	Formatting   string        `bson:"formatting,omitempty"`
-	Labels       []string      `bson:"labels,omitempty"`
+	Id            bson.ObjectId `bson:"_id,omitempty"`
+	Uuid          string        `bson:"uuid"`
+	Created       int64         `bson:"created"`
+	Description   string        `bson:"description,omitempty"`
+	Modified      int64         `bson:"modified"`
+	Origin        int64         `bson:"origin"`
+	Name          string        `bson:"name"`
+	Min           interface{}   `bson:"min,omitempty"`
+	Max           interface{}   `bson:"max,omitempty"`
+	DefaultValue  interface{}   `bson:"defaultValue,omitempty"`
+	Type          string        `bson:"type,omitempty"`
+	UomLabel      string        `bson:"uomLabel,omitempty"`
+	Formatting    string        `bson:"formatting,omitempty"`
+	Labels        []string      `bson:"labels,omitempty"`
+	MediaType     string        `bson:"mediaType"`
+	FloatEncoding string        `bson:"floatEncoding"`
```

Comment:
@cloudxxx8 It looks like these need to have the `omitempty` attribute to conform with the other string types above.

Reply:
@tsconn23 sorry, I missed them. Just added.  Thanks for the review